### PR TITLE
Relax `faraday` dep

### PR DESCRIPTION
**Pull request by andreibondarev**

### CODE REVIEW

File: qdrant.gemspec [hunk 1]
```diff
@@ -35,6 +35,7 @@ Gem::Specification.new do |spec|
   # For more information and examples about making a new gem, check out our
   # guide at: https://bundler.io/guides/creating_gem.html
 
-  spec.add_dependency "faraday", "~> 2.7"
+  spec.add_dependency "faraday", "~> 1"
```
Comment (rthbound) [inline qdrant.gemspec:38]: Seems like adding faraday_middleware may prevent installing faraday 2.0.1 or higher.

Is there a reason this gem needs to keep support for versions less than 2.0? Could we use `~> 2.0.1` and drop `faraday_middleware` as a dependency?

Comment (andreibondarev) [reply qdrant.gemspec:38]: Yeah I think the move is to bump it to faraday >2 everywhere. When I say "everywhere" I mean all of the other gems also required by langchainrb.

Comment (rthbound) [reply qdrant.gemspec:38]: Some are already allowing for 2.0.1 and higher:

- replicate-ruby:  ">= 1.0"
- hugging-face: ">= 1.0"
- anthropic: ">= 1"
- ruby-openai: ">= 1"

This one had an outstanding pull request already:

- open-weather-ruby-client: [outstanding PR](https://github.com/dblock/open-weather-ruby-client/pull/33)

Created MRs for these:

- cohere-ruby: https://github.com/andreibondarev/cohere-ruby/pull/6
- google_palm_api: https://github.com/andreibondarev/google_palm_api/pull/4
- milvus: https://github.com/andreibondarev/milvus/pull/5
- qdrant-ruby: https://github.com/andreibondarev/qdrant-ruby/pull/4
- weaviate-ruby: https://github.com/andreibondarev/weaviate-ruby/pull/31

Comment (andreibondarev) [reply qdrant.gemspec:38]: @rthbound I've released new versions for all of these gems. Would you like to try bumping up the dependencies on Langchain.rb?

Comment (rthbound) [reply qdrant.gemspec:38]: Looks like we'll still need to wait for the `open-weather-ruby-client` gem to merge before we can update Langchain.rb's gemspec to specify the latest versions of these gems. If we update sooner, then `bundle install` won't run successfully within the langchain project.

Comment (andreibondarev) [reply qdrant.gemspec:38]: @rthbound Looks like we're all set!

Comment (andreibondarev) [reply qdrant.gemspec:38]: @rthbound I went ahead and bumped the gems versions: https://github.com/andreibondarev/langchainrb/pull/302

**Pull request merged**